steel_industry/functions.py

```diff
@@ -5,11 +5,14 @@
 
 
 def split_name(name):
-    '''
-    :param:  name: (string) - Entry in the “name” column of result.csv
-    :return: (list) of strings
-    :logik: divide a "_"-seperated string into a list of strings
-    '''
+    """
+    divide a "_"-seperated string into a list of strings, depending on last charakter is digit.
+    return: list of strings.
+    Parameters
+    ----------
+    name: string
```

Comment:
```suggestion
    string
```
denn genau genommen gibt die funktion nur einen Teil von `name` zurück. Und ich glaube sie gibt eine list zurück, oder?

Reply:
split_name gibt eine Liste zurück. Das ist korrekt. Aber alle Elemente werden erhalten. Die Zahl am Ende (--1) wird vorher abgeschnitten.